Remotion/Web/ClientScript/Scripts/DropDownMenu.ts

```diff
@@ -756,21 +770,19 @@ class DropDownMenu
     }
   }
 
-  private static Options_OnKeyDown(event: JQueryKeyEventObject, dropDownMenu: JQuery): void
+  private static Options_OnKeyDown(event: KeyboardEvent, dropDownMenu: HTMLElement): void
   {
-    ArgumentUtility.CheckNotNullAndTypeIsJQuery('dropDownMenu', dropDownMenu);
-
     if (DropDownMenu._currentPopup == null)
       return;
 
-    var itemInfos = DropDownMenu._menuInfos[dropDownMenu[0].id]!.ItemInfos!;
+    var itemInfos = DropDownMenu._menuInfos[dropDownMenu.id]!.ItemInfos!;
     if (itemInfos.length === 0)
       return;
 
     var oldIndex;
     var isSelectionUpdated = false;
-    var dropDownMenuItems = $(DropDownMenu._currentPopup).find('ul').children();
-    var currentItemIndex = $('.' + DropDownMenu._itemSelectedClassName, DropDownMenu._currentPopup).index();
+    var dropDownMenuItems = Array.from(DropDownMenu._currentPopup.querySelectorAll('ul')).flatMap((el: HTMLUListElement) => Array.from(el.children));
```

Comment:
Flatmap takes an enumerable so we don't have to convert to array first if I am not mistaken.

```suggestion
    var dropDownMenuItems = Array.from(DropDownMenu._currentPopup.querySelectorAll('ul')).flatMap(el => el.children);
```

Reply:
Getting errors without it...

Reply:
Interesting because my IDE does not show me an error and I also get no error when testing in the TypeScript playground. Did it not compile or just show a red squigglies?

Reply:
Did not compile. Please check it out, have a look. I'm confused.

Reply:
Ok my bad, we need to do the `Array.from`, flatMap also supports returning only one element instead of an array (aka a map). This is what happens if we do not use the `Array.from` it maps to a `NodeList[]` instead of applying the flatmap